lua/lspconfig/server_configurations/ungrammar_languageserver.lua

```diff
@@ -0,0 +1,36 @@
+local util = require 'lspconfig.util'
+
+return {
+  default_config = {
+    cmd = { 'ungrammar-languageserver', '--stdio' },
+    filetypes = { 'ungrammar' },
+    root_dir = function(fname)
+      return util.root_pattern '.git'(fname)
+    end,
```

Comment:
```suggestion
    root_dir = util.find_git_ancestor, 
```

Reply:
> 

Thank @glepnir, I have force-pushed to avoid the noise commits.

Reply:
don't we want to discourage `find_git_ancestor` ?

Reply:
can be handled uniformly during subsequent cleanup. this is relatively simple at present.  plann to stop recommending this function syntax sugar for all new PRs now?

Reply:
I have create #3313 then there can use `root_dir = vim.fs.root(0, '.git')`